AzureTable/Microsoft.DataTransfer.AzureTable/RemoteLogging/RemoteLogging.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.CosmosDB;
+using Microsoft.Azure.CosmosDB.Table;
+using Microsoft.Azure.Storage;
+
+namespace Microsoft.DataTransfer.AzureTable.RemoteLogging
+{
+    /// <summary>
+    /// Class to support remote logging in CosmosDB Tables
+    /// </summary>
+    public sealed class RemoteLogging : IRemoteLogging
+    {
+        private static CloudTable migrationLogger;
+        private readonly int tableThroughput = 20000;
+        private readonly string tableName = "migrationLogs";
+
+        /// <summary>
+        /// Remote logger constructor
```

Comment:
I think we should follow some standard for constructor xmldocs. Something enforced by StyleCop will do: http://stylecop.soyuz5.com/SA1642.html

Reply:
done with new iteration.